troubleshooting/known_issues/cray-console-node_pods_in_CrashLoopBackOff.md

```diff
@@ -0,0 +1,90 @@
+# `cray-console-node` pods in `CrashLoopBackOff`
+
+## Issue description
+
+After installing CSM 1.6, `cray-console-node` Kubernetes pods intermittently go into a `CrashLoopBackOff` state.
+This happens because of a permission issue with the `/var/log/conman` directory inside the container.
```

Comment:
```suggestion
The `cray-console-node` Kubernetes pods may go into a `CrashLoopBackOff` state.
This happens because of a permission issue with the `/var/log/conman` directory inside the container.
```